strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/cron/jobs/RebuildMavenIndexesCronJobTestIT.java

```diff
@@ -141,21 +128,9 @@ public void testRebuildIndexesInRepository(@MavenRepository(repositoryId = REPOS
             {
                 try
                 {
-                    String query1 = String.format("+g:%s +a:%s +v:%s +p:jar", GROUP_ID, ARTIFACT_ID1, VERSION);
-                    SearchRequest request1 = new SearchRequest(STORAGE0,
-                                                               repository.getId(),
-                                                               query1,
-                                                               MavenIndexerSearchProvider.ALIAS);
-
-                    assertTrue(artifactSearchService.contains(request1));
-
-                    String query2 = String.format("+g:%s +a:%s +v:%s +p:jar", GROUP_ID, ARTIFACT_ID2, VERSION);
-                    SearchRequest request2 = new SearchRequest(STORAGE0,
-                                                               repository.getId(),
-                                                               query2,
-                                                               MavenIndexerSearchProvider.ALIAS);
-
-                    assertTrue(artifactSearchService.contains(request2));
+                    RepositoryPath indexPath = repositoryIndexDirectoryPathResolver.resolve(repository);
+                    RepositoryPath packedIndexPath = indexPath.resolve(IndexingContext.INDEX_FILE_PREFIX + ".gz");
+                    Assertions.assertThat(packedIndexPath).matches(Files::exists);
```

Comment:
Cosmetic: Would you mind using static import statements for these assertion methods? Thanks! :)

Reply:
done

Reply:
Thanks! :)